examples/scrollable/src/main.rs

```diff
@@ -50,209 +66,333 @@ impl Application for ScrollableDemo {
+                .height(Length::Fill)
+                .horizontal_scroll(
+                    scrollable::Horizontal::new()
+                        .scrollbar_height(self.scrollbar_width)
+                        .scrollbar_margin(self.scrollbar_margin)
+                        .scroller_height(self.scroller_width),
+                )
+                .style(theme::Scrollable::Custom(Box::new(
+                    ScrollbarCustomStyle,
+                )))
+                .id(SCROLLABLE_ID.clone())
+                .on_scroll(Message::Scrolled),
+                Direction::Multi => scrollable(
+                    //horizontal content
+                    row![
+                        column![
+                            text("Let's do some scrolling!"),
+                            vertical_space(Length::Units(2400))
+                        ],
+                        scroll_to_end_button(scrollable::Direction::Horizontal),
+                        text("Horizontal - Beginning!"),
+                        horizontal_space(Length::Units(1200)),
+                        //vertical content
+                        column![
+                            text("Horizontal - Middle!"),
+                            scroll_to_end_button(
+                                scrollable::Direction::Vertical
+                            ),
+                            text("Vertical - Beginning!"),
+                            vertical_space(Length::Units(1200)),
+                            text("Vertical - Middle!"),
+                            vertical_space(Length::Units(1200)),
+                            text("Vertical - End!"),
+                            scroll_to_beginning_button(
+                                scrollable::Direction::Vertical
+                            )
+                        ]
+                        .align_items(Alignment::Fill)
+                        .spacing(40),
+                        horizontal_space(Length::Units(1200)),
+                        text("Horizontal - End!"),
+                        scroll_to_beginning_button(
+                            scrollable::Direction::Horizontal
+                        ),
+                    ]
+                    .align_items(Alignment::Center)
+                    .padding([0, 40, 0, 40])
+                    .spacing(40),
+                )
+                .height(Length::Fill)
+                .scrollbar_width(self.scrollbar_width)
+                .scrollbar_margin(self.scrollbar_margin)
+                .scroller_width(self.scroller_width)
+                .horizontal_scroll(
+                    scrollable::Horizontal::new()
+                        .scrollbar_height(self.scrollbar_width)
+                        .scrollbar_margin(self.scrollbar_margin)
+                        .scroller_height(self.scroller_width),
+                )
+                .style(theme::Scrollable::Custom(Box::new(
+                    ScrollbarCustomStyle,
+                )))
```

Comment:
May be a good idea to add a `custom` helper to `theme::Scrollable` that does the boxing internally so the resulting `view` code is cleaner.